pages/index.tsx

```diff
@@ -81,35 +79,57 @@ const Home: NextPage = () => {
             // TODO: Replace scroll overflow for a modal.
             style={{ maxHeight: "25vh", overflow: "auto" }}
             extra={
-              <Button
-                type="dashed"
-                disabled={!!isSaving}
-                icon={iconToRender}
-                onClick={() => {
-                  setIsSaving(article._id);
-                  const starredArticlesToSet = new Set(starredArticles);
-                  if (starredArticlesToSet?.has(article._id)) {
-                    starredArticlesToSet.delete(article._id);
-                  } else {
-                    starredArticlesToSet?.add(article._id);
-                  }
-                  fetch("/api/user.update", {
-                    method: "POST",
-                    body: JSON.stringify({
-                      starredArticles: [...starredArticlesToSet!]
-                    })
-                  })
-                    .then((res) => res.json())
-                    .then((res: UserType) => {
-                      setStarredArticles(
-                        new Set(res.starredArticles.map(({ _id }) => _id))
-                      );
-                    })
-                    .finally(() => {
-                      setIsSaving(undefined);
-                    });
-                }}
-              />
+              <Row gutter={[5,0]}>
+                <Col span={8}>
+                  <Button
+                    type="primary"
+                    icon={<ArrowsAltOutlined/>}
+                    onClick={() => {
+                      router.push(`/articles/${article._id}`);
+                    }}
+                  />
+                </Col>
+                <Col span={8}>
+                  <Button
+                    icon={<ShareAltOutlined />}
+                    onClick={() => {
+                      navigator.clipboard.writeText(`${window.location.host}/articles/${article._id}`);
```

Comment:
Chingón que lo encontraste!

Easter egg: el clipboard solo funciona en contextos seguros HTTPS, si alguna vez sirves tu app sobre HTTP, [`.clipboard` es undefined](https://stackoverflow.com/questions/52054635/copy-clipboard-function-working-locally-but-not-working-server).

Reply:
Qué buen dato!! thx!